benchmarks/operator_benchmark/benchmark_pytorch.py

```diff
@@ -17,7 +17,28 @@
 def PyTorchOperatorTestCase(test_name, op_type, input_shapes, op_args, run_mode):
     """Benchmark Tester function for Pytorch framework.
     """
-    inputs = [torch.from_numpy(benchmark_utils.numpy_random_fp32(*input)) for input in input_shapes]
+    inputs = []
+    is_contig = 'contig' not in op_args or op_args['contig']
+    dtype = op_args['dtype'] if 'dtype' in op_args else torch.float32
+    for shape in input_shapes:
+        tensor_shape = list(shape)
+        if not is_contig:
+            tensor_shape = [s * 2 for s in tensor_shape]
+        if dtype in [torch.float32, torch.float64]:
```

Comment:
dtype.is_floating_point

Reply:
I actually want to avoid `float16` because `torch.rand` doesn't work with it, that's why there's explicit check for `torch.float32` and `torch.float64` only